src/app/public/modules/file-attachment/fixtures/file-drop-control-no-form.component.fixture.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Component,
+  ViewChild
+} from '@angular/core';
+
+import {
+  SkyFileDropControlComponent
+} from '../file-drop-control.component';
+
+@Component({
+  selector: 'file-drop-control-no-form-test',
+  templateUrl: './file-drop-control-no-form.component.fixture.html'
+})
+export class FileDropControlNoFormTestComponent {
+
+  public required: boolean = false;
```

Comment:
It looks like `required` and `showLabel` aren't being used?

Reply:
Removed